app/assets/stylesheets/views/_browse.scss

```diff
@@ -26,28 +16,10 @@ $browse-header-background-colour: #263135;
   overflow: auto;
 }
 
-.browse__breadcrumb-border {
-  background: govuk-colour("blue");
-  height: 10px;
-}
-
-.global-bar-present {
-  .browse__breadcrumb-border {
-    height: 0;
-  }
-}
-
 // The header spacings are a work in progress.
 .browse__header-wrapper {
   padding-bottom: govuk-spacing(7);
 
-  // A tiny white seam sometimes appears, despite there being no gap between
-  // this element and the previous element. This shifts the element up by a
-  // pixel to prevent the seam from appearing and adds padding to reposition the
-  // content.
-  border-top: 1px solid $browse-header-background-colour;
-  margin-top: -1px;
-
```

Comment:
Good spot, agree that it looks safe to remove this now, I tested in several browsers, including Chrome, Firefox, Safari and Safari on iOS, and could not see any issues.